test/unittests/instantiate_test.cpp

```diff
@@ -392,8 +410,8 @@ TEST(instantiate, DISABLED_imported_globals_mismatched_type)
     const auto bin2 = from_hex("0061736d01000000020b01036d6f64026731037f00");
     const auto module2 = parse(bin2);
 
-    EXPECT_THROW_MESSAGE(
-        instantiate(module2, {}, {}, {}, {*g}), instantiate_error, "type mismatch");
+    EXPECT_THROW_MESSAGE(instantiate(module2, {}, {}, {}, {*g}), instantiate_error,
+        "global 0 value type doesn't match module's global type");
```

Comment:
Modify or and a test where non-0 global is wrong.

Reply:
Added a case to `imported_globals_mismatched_type` test